ctapipe/conftest.py

```diff
@@ -62,6 +62,42 @@ def test_my_thing(test_event):
     return deepcopy(_global_example_event)
 
 
+@pytest.fixture(scope="function")
+def gamma_off_axis_500_gev():
```

Comment:
I suggest naming this `gamma_event_off_axis_500_gev`, so it's clear that this fixtures is an event, not some other information. 

It's a but long, but even `subarray_and_gamma_event_off_axis_500_gev` would be more clear, as it's not just an event, but also the subarray description...